<?xml version="1.0" encoding="UTF-8"?>
<testrun name="test" project="CopyAssistant" tests="4" started="4" failures="1" errors="0" ignored="0">
  <testsuite name="com.sgnn7.cpassistant.CopierTest" time="0.027">
    <testcase name="folderThatIsAddedGetsExpandedToIncludeAllFiles" classname="com.sgnn7.cpassistant.CopierTest" time="0.007"/>
    <testcase name="invokingCopyCopiesAllFilesToTarget" classname="com.sgnn7.cpassistant.CopierTest" time="0.011"/>
    <testcase name="invokingCopyWithABadFileCopiesAllFilesToTargetAndReturnsFailedOnes" classname="com.sgnn7.cpassistant.CopierTest" time="0.004"/>
    <testcase name="invokingCopyWithABadFolderDoesNotTryToCopyFilesIntoThatFolder" classname="com.sgnn7.cpassistant.CopierTest" time="0.004">
      <failure>java.lang.AssertionError: File 'data\blah\abc.txt' should not have been copied
	at org.junit.Assert.fail(Assert.java:91)
	at com.sgnn7.cpassistant.CopierTest$MockCopier.copyFile(CopierTest.java:124)
	at com.sgnn7.cpassistant.Copier.beginCopying(Copier.java:83)
	at com.sgnn7.cpassistant.CopierTest.invokingCopyWithABadFolderDoesNotTryToCopyFilesIntoThatFolder(CopierTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:73)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:46)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:46)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
  </testsuite>
</testrun>
